opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/SpanBuilderShim.java

```diff
@@ -77,8 +67,17 @@ public SpanBuilder asChildOf(Span parent) {
     return this;
   }
 
+  @Override
+  public SpanBuilder asChildOf(SpanContext parent) {
+    return addReference(null, parent);
+  }
+
   @Override
   public SpanBuilder addReference(String referenceType, SpanContext referencedContext) {
+    if (referencedContext == null) {
+      return this;
+    }
+
     // TODO - Use referenceType
```

Comment:
Is this comment still actual? I'm not sure what it means

Reply:
Nope, this is not actual anymore - will be cleaning up comments in another (small) commit ;)